pymoab-visit-py2-18.04

```diff
@@ -0,0 +1,28 @@
+FROM svalinn/pymoab-py2-18.04:latest
+
+ENV HOME /root
```

Comment:
Is that not already set in the previous container ?

Reply:
yes.. will remove